packages/appium/docs/en/guides/log-filters.md

```diff
@@ -2,4 +2,77 @@
 title: Log Filtering
 ---
 
-TODO
+Sometimes it might be necessary to hide sensitive information, like passwords, device identifiers,
+hashes, etc..., from the server log. Appium makes it possible to ensure such information is
+redacted in logs via the `--log-filters` command line argument. This argument allows you to provide
+the path to a special config file, containing one or more log obfuscation rules.
+
+## Config Format
```

Comment:
should probably add a note/link that this can be configured in an appium config file (without an external JSON file)

Reply:
ah good idea

Reply:
@boneskull does it look correct now?